core/plugins/org.polarsys.capella.core.semantic.queries/plugin.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <?eclipse version="3.2"?>
 <!--
-  Copyright (c) 2018, 2020 THALES GLOBAL SERVICES.
+  Copyright (c) 2018, 2022 THALES GLOBAL SERVICES.
```

Comment:
Copyright (c) 2018, 2022 THALES GLOBAL SERVICES and others.

+ contributors - summary/scope of the change/ticket id